test/common/tls/test_data/certs.sh

```diff
@@ -16,6 +16,73 @@ cleanup() {
     rm ./intermediate_crl_*
 }
 
+
+# $@=<trust domain, CA cert, sequence number> (repeatable for multiple domains and certs per domain)
+generate_spiffe_trust_bundle_mapping() {
+    local trust_domains=()
+
+    # Collecting all trust domain arguments
+    while [[ $# -gt 0 ]]; do
+        trust_domains+=( "$1" "$2" "$3" )
+        shift 3
+    done
+
+    # Use a single redirect for the entire block
+    {
+        echo "{"
+        echo "  \"trust_domains\": {"
+
```

Comment:
I think I have a comment for this. Couldn't all these be moved to `test/extensions/transport_sockets/tls/cert_validator/spiffe`?